File: University/ZarbeMatrixHaS1.py
def main():
    row1, column, column2 = map(int, input().split())
    array1 = [[0 for _ in range(101)] for _ in range(101)]
    array2 = [[0 for _ in range(101)] for _ in range(101)]
    result = [[0 for _ in range(101)] for _ in range(101)]

    for i in range(row1):
        array1[i] = list(map(int, input().split()))

    for i in range(column):
        array2[i] = list(map(int, input().split()))

    for i in range(row1):
        for j in range(column2):
            for k in range(column):
                result[i][j] += array1[i][k] * array2[k][j]

    for i in range(row1):
        for j in range(column2):
            print(result[i][j], end=" ")
        print("")

if __name__ == "__main__":
    main()

File: University/ZarbeMatrixHaS1.cs
using System;
using System.IO;
public class Main
{
    public static void Main(string[] args)
    {
        var input =  "Inputs";
        var row1 = input.nextByte();
        var column = input.nextByte();
        var column2 = input.nextByte();
        var array1 = new int[101,101];
        var array2 = new int[101,101];
        var result = new int[101,101];
        for (byte i = 0; i < row1; i++)
        {
            for (byte j = 0; j < column; j++)
            {
                array1[i,j] = Convert.ToInt64(Console.ReadLine());
            }
        }
        for (byte i = 0; i < column; i++)
        {
            for (byte j = 0; j < column2; j++)
            {
                array2[i,j] = Convert.ToInt64(Console.ReadLine());
            }
        }
        for (int i = 0; i < row1; i++)
        {
            for (int j = 0; j < column2; j++)
            {
                for (int k = 0; k < column; k++)
                {
                    result[i,j] += array1[i,k] * array2[k,j];
                }
            }
        }
        for (byte i = 0; i < row1; i++)
        {
            for (byte j = 0; j < column2; j++)
            {
                Console.Write(result[i,j].ToString() + " ");
            }
            Console.WriteLine("");
        }
    }
}
